sensor/common/image/service_impl_test.go

```diff
@@ -0,0 +1,280 @@
+package image
+
+import (
+	"context"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/pkg/errors"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/internalapi/sensor"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/concurrency"
+	cacheMocks "github.com/stackrox/rox/pkg/expiringcache/mocks"
+	"github.com/stackrox/rox/sensor/common"
+	imageMocks "github.com/stackrox/rox/sensor/common/image/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestImageService(t *testing.T) {
+	suite.Run(t, new(imageServiceSuite))
+}
+
+type imageServiceSuite struct {
+	suite.Suite
+	mockCtrl          *gomock.Controller
+	mockCache         *cacheMocks.MockCache
+	mockRegistryStore *imageMocks.MockregistryStore
+	mockCentral       *imageMocks.MockcentralClient
+	mockLocalScan     *imageMocks.MocklocalScan
+	service           *serviceImpl
+}
+
+var _ suite.SetupTestSuite = (*imageServiceSuite)(nil)
+
+func (suite *imageServiceSuite) SetupTest() {
+	suite.mockCtrl = gomock.NewController(suite.T())
+	suite.mockCache = cacheMocks.NewMockCache(suite.mockCtrl)
+	suite.mockRegistryStore = imageMocks.NewMockregistryStore(suite.mockCtrl)
+	suite.mockCentral = imageMocks.NewMockcentralClient(suite.mockCtrl)
+	suite.mockLocalScan = imageMocks.NewMocklocalScan(suite.mockCtrl)
+	suite.service = &serviceImpl{
+		imageCache:    suite.mockCache,
+		registryStore: suite.mockRegistryStore,
+		localScan:     suite.mockLocalScan,
+		centralReady:  concurrency.NewSignal(),
+		centralClient: suite.mockCentral,
+	}
+}
+
+func (suite *imageServiceSuite) TestGetImage() {
```

Comment:
```suggestion
func (s *imageServiceSuite) TestGetImage() {
```

Use single letter for receivers

Reply:
Done